app/hotels/src/allHotels/RenderResults.js

```diff
@@ -0,0 +1,124 @@
+// @flow strict
+
+import * as React from 'react';
+import { Animated, View } from 'react-native';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import { StyleSheet, AdaptableLayout } from '@kiwicom/mobile-shared';
+
+import MapScreen from '../map/allHotels/MapScreen';
+import AllHotelsSearchList from './AllHotelsSearchList';
+import type { RenderResults as RenderResultsType } from './__generated__/RenderResults.graphql';
+import SearchResultsContext, {
+  type ResultType,
+} from '../navigation/allHotels/SearchResultsContext';
+
+type PropsWithContext = {|
+  ...Props,
+  +show: ResultType,
+|};
+
+const topValue = 1000;
+const lowValue = -100;
+
+class RenderResults extends React.Component<PropsWithContext> {
+  mapAnimation = new Animated.Value(topValue);
```

Comment:
Are you instantiating these class properties twice? Once here and once below in the constructor?